docs/helm/helm_create.md

```diff
@@ -30,6 +30,13 @@ do not exist, Helm will attempt to create them as it goes. If the given
 destination exists and there are files in that directory, conflicting files
 will be overwritten, but other files will be left alone.
 
+Its aim is to show a basic chart with resources that can be installed to demonstrate
+e2e capability. It is not designed to show all resource types.
+
+If you want to create daemonset or statefulset, here is Kubernetes layer knowledge,
```

Comment:
We want to describe what the scaffold chart provides. The scaffold chart is to provide an e2e example and is not meant to cover all Kubernetes resources. The Kubernetes documentation covers that and does mot need to be listed here.